roles/kubernetes/node/tasks/main.yml

```diff
@@ -70,9 +70,12 @@
 
 - name: Verify br_netfilter module path exists
   file:
-    path: /etc/modules-load.d
+    path: "{{ item }}"
     state: directory
     mode: 0755
+  loop:
+    - /etc/modules-load.d
+    - /etc/modprobe.d
```

Comment:
HI [@cosandr](https://github.com/cosandr)
If the bug only effect the Flat,
It's better to use the "if os == Flatcat", and with comments on the code

Reply:
Agreed